### PR TITLE
Feature/library

### PR DESCRIPTION
**Pull request by ninjavaz**
Next changes from previous PR has been made

### CODE REVIEW

File: src/ThingooConnector.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		String access_token = doc["access_token"];
+		return access_token;
+	}
+	else
+	{
+		// Serial.printf("[HTTP] POST... failed, error: %s\n", http.errorToString(http_response_code).c_str());
+		String exception = http.errorToString(http_response_code);
+		Serial.println(exception); 
+		
+		// It doesnt work, nothing printing, nothing returning, implemented well as library says -
```
Comment (ninjavaz) [inline src/ThingooConnector.cpp:59]: .

Comment (BartlomiejRasztabiga) [reply src/ThingooConnector.cpp:59]: We surely don't want to return an exception. Just throw some custom exception to be caught by the library user

File: examples/main.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#include "ThingooConnector.h"
+
+#include <ESP8266HTTPClient.h>
+#include <ArduinoJson.h>
+#include <WiFiManager.h>
+#include <WiFiClientSecureBearSSL.h>
+
+const char* fingerprint = "68 E7 10 AD 43 DE EE 7E 24 67 71 BC 43 77 16 23 78 64 AF X5";
```
Comment (BartlomiejRasztabiga) [inline examples/main.cpp]: Remove our instance details, this is a public library. Use some fake data

**Review by BartlomiejRasztabiga — CHANGES_REQUESTED** [review]
2 changes and it's ready to be published! :tada:

File: examples/main.cpp (hunk 2 above)
Comment (ninjavaz) [reply examples/main.cpp]: It s a fake data :D

Comment (BartlomiejRasztabiga) [reply examples/main.cpp]: device_id, host and secret_key surely aren't ;)

**Pull request closed without merging**